### ISSUE TITLE
Building SuperLU_DIST tests fail

### ISSUE THREAD

**Issue by johannesring** [hashdist/hashstack#715]
Maximillian Behr [reported](http://fenicsproject.org/pipermail/fenics-support/2015-April/001434.html) problems building SuperLU_DIST with the [FEniCS install script](https://bitbucket.org/fenics-project/fenics-developer-tools/src/561ed5e069408ce12bf29cf0f21bd750f6c9381e/install/fenics-install.sh?at=master).

From the [build.log](https://gist.github.com/johannesring/08aeb43e6026644b3e1e) we see that it fails to build the tests:

    ...
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.o: In function `main':
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x15c): undefined reference to `zreadhb_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x29c): undefined reference to `zCreate_CompCol_Matrix_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x2a8): undefined reference to `doublecomplexMalloc_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x2bc): undefined reference to `doublecomplexMalloc_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x2d0): undefined reference to `doublecomplexMalloc_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x300): undefined reference to `zGenXtrue_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x324): undefined reference to `zFillRHS_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x410): undefined reference to `pzgssvx_ABglobal'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x47c): undefined reference to `pzgssvx_ABglobal'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x648): undefined reference to `zallocateA_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x718): undefined reference to `zinf_norm_error_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x780): undefined reference to `pzgssvx_ABglobal'
    2015/04/01 12:38:34 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x7b4): undefined reference to `zinf_norm_error_dist'
    2015/04/01 12:38:34 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
    2015/04/01 12:38:34 - INFO: [package:run_job] make: *** [pzdrive1_ABglobal] Error 1
    2015/04/01 12:38:34 - INFO: [package:run_job] make: *** Waiting for unfinished jobs.... 
    ...

**Comment by vincentkersten** [comment]
:+1:  , here the same 
````
2015/03/18 08:12:03 - INFO: [package:run_job] /Scratch/vincent/fenics.stable/fenics/bld/mpi/hbd47qrzmwye/bin/mpicc  pddrive1_ABglobal.o sp_ienv.o /Scratch/vincent/fenics.stable/fenics/tmp/superlu_dist-dkkgllj66sqt/lib/libsuperlu_dist.a -Wl,-rpath,/Scratch/vincent/fenics.stable/fenics/bld/lapack/zzrggbkqot7x/lib -L/Scratch/vincent/fenics.stable/fenics/bld/lapack/zzrggbkqot7x/lib -lopenblas -Wl,-rpath,/Scratch/vincent/fenics.stable/fenics/bld/lapack/zzrggbkqot7x/lib -L/Scratch/vincent/fenics.stable/fenics/bld/lapack/zzrggbkqot7x/lib -lopenblas -Wl,-rpath,/Scratch/vincent/fenics.stable/fenics/bld/parmetis/swux5owl6onc/lib -L/Scratch/vincent/fenics.stable/fenics/bld/parmetis/swux5owl6onc/lib -lparmetis -Wl,-rpath,/Scratch/vincent/fenics.stable/fenics/bld/parmetis/swux5owl6onc/lib -L/Scratch/vincent/fenics.stable/fenics/bld/parmetis/swux5owl6onc/lib -lmetis -lm -o pddrive1_ABglobal
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0xf9): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x23f): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x248): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x25d): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x272): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x2aa): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x2d4): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x3ac): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x428): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x5dd): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x605): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x676): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1_ABglobal.c:(.text.startup+0x6a5): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive1_ABglobal] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** Waiting for unfinished jobs....
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0xfc): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0x204): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0x240): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0x2be): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0x2ef): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0x441): undefined reference to `zSolveFinalize'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0x484): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive1.c:(.text.startup+0x502): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.o: In function `zcreate_matrix':
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x86): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1c4): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1d4): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1f2): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x21e): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x251): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x330): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x458): undefined reference to `zCreate_CompRowLoc_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x460): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x51f): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x65f): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive1] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x103): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x1b8): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x2f4): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x32c): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x3c2): undefined reference to `zCreate_CompRowLoc_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x41c): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x45a): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3.c:(.text.startup+0x5a9): undefined reference to `zSolveFinalize'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.o: In function `zcreate_matrix':
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x86): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1c4): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1d4): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1f2): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x21e): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x251): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x330): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x458): undefined reference to `zCreate_CompRowLoc_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x460): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x51f): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x65f): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive3] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive.c:(.text.startup+0x1af): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive.c:(.text.startup+0x1e1): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive.c:(.text.startup+0x32d): undefined reference to `zSolveFinalize'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.o: In function `zcreate_matrix':
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x86): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1c4): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1d4): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1f2): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x21e): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x251): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x330): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x458): undefined reference to `zCreate_CompRowLoc_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x460): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x51f): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x65f): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x212): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x2d1): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x2da): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x2ef): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x323): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x346): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x3f6): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x522): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x5e1): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x5ea): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x5ff): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x633): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x656): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x706): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x830): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x8ff): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x9bd): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4_ABglobal.c:(.text.startup+0x9eb): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive4_ABglobal] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0xfc): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x248): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x251): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x266): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x29e): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x2c8): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x2d1): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x354): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x46c): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x502): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x54a): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x6f0): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x718): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x7a3): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x7eb): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive3_ABglobal.c:(.text.startup+0x81a): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive3_ABglobal] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x103): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x1b8): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x2f4): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x32c): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x3d1): undefined reference to `zCreate_CompRowLoc_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x42b): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x469): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2.c:(.text.startup+0x5b9): undefined reference to `zSolveFinalize'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.o: In function `zcreate_matrix':
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x86): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1c4): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1d4): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1f2): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x21e): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x251): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x330): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x458): undefined reference to `zCreate_CompRowLoc_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x460): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x51f): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x65f): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive2] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4.c:(.text.startup+0x288): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4.c:(.text.startup+0x2c4): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4.c:(.text.startup+0x51f): undefined reference to `pzgssvx'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4.c:(.text.startup+0x55b): undefined reference to `pzinf_norm_error'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive4.c:(.text.startup+0x5a6): undefined reference to `zSolveFinalize'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.o: In function `zcreate_matrix':
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x86): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1c4): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1d4): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x1f2): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x21e): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x251): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x330): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x458): undefined reference to `zCreate_CompRowLoc_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x460): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x51f): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] zcreate_matrix.c:(.text+0x65f): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive4] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0xf0): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x222): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x22b): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x24d): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x281): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x2a9): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x353): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x50d): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive_ABglobal.c:(.text.startup+0x53b): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive_ABglobal] Error 1
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.o: In function `main':
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0xfd): undefined reference to `zreadhb_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x241): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x24a): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x26c): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x2a4): undefined reference to `zGenXtrue_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x2ce): undefined reference to `zFillRHS_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x2d7): undefined reference to `doublecomplexMalloc_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x355): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x470): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x515): undefined reference to `zCreate_CompCol_Matrix_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x55f): undefined reference to `pzgssvx_ABglobal'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x717): undefined reference to `zallocateA_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x74b): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] pzdrive2_ABglobal.c:(.text.startup+0x773): undefined reference to `zinf_norm_error_dist'
2015/03/18 08:12:03 - INFO: [package:run_job] collect2: error: ld returned 1 exit status
2015/03/18 08:12:03 - INFO: [package:run_job] make: *** [pzdrive2_ABglobal] Error 1
2015/03/18 08:12:04 - ERROR: [package:run_job] Command '[u'/bin/bash', '_hashdist/build.sh']' returned non-zero exit status 2
2015/03/18 08:12:04 - ERROR: [package:run_job] command failed (code=2); raising
````

**Comment by maxbehr90** [comment]
i set the number of threads in fenic-install.sh to 1 that solved my problem. 
besides i have a ppc64 power7 big endian machine which is not supported by openblas, 
is it possible to change *.yaml file to force hashstack using atlas instead of ?

**Comment by vincentkersten** [comment]
I have the PROCS on 48 , will try with 1, thanks for the hint

**Comment by johannesring** [comment]
OK, so removing the `-j` option to `make` when building the tests would probably help. I will make a pull request.

**Comment by johannesring** [comment]
I could reproduce this when building with `hit build -j48`. Removing the `-j` option when building the tests did not fix the problem. I also had to remove it when building the library. I will update #716.

**Comment by johannesring** [comment]
Fixed in #716.
